requirements/requirements.txt

```diff
@@ -10,3 +10,4 @@ scikit-learn
 omegaconf>=2.0.2
 hydra-core>=1.0.3
 transformers>=3.1.0
+nvidia-eff @ http://sqrl.nvidia.com/dldata/pip-scratch/nvidia-eff/nvidia_eff-0.1.0-py3-none-any.whl
```

Comment:
Please remove this from requirements. Running `pip install -r requirement.txt` results in a loop
```
Collecting nvidia-eff@ http://sqrl.nvidia.com/dldata/pip-scratch/nvidia-eff/nvidia_eff-0.1.0-py3-none-any.whl
  WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPConnection object at 0x7f4be827b8d0>: Failed to establish a new connection: [Errno -2] Name or service not known',)': /dldata/pip-scratch/nvidia-eff/nvidia_eff-0.1.0-py3-none-any.whl
  WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPConnection object at 0x7f4be827bcc0>: Failed to establish a new connection: [Errno -2] Name or service not known',)': /dldata/pip-scratch/nvidia-eff/nvidia_eff-0.1.0-py3-none-any.whl
```

Reply:
hmmm, ok, this is bad.

I was using `pip install -e .`, also tested `reinstall.sh`, worked properly.

Let me try to replicate that in a new environment.

Reply:
I don't think this file is accessible outside the internal network, so please retest when not on the company VPN

Reply:
Hmmmm, not sure about that, there are plenty of tools (including DALI) that are distributed this way.
@DEKHTIARJonathan can you please comment on this? 

Independently, let me test that "outside" of NVIDIA network...

Reply:
@blisc you are right! I just tried to reinstall NeMo on my laptop (outside of VPN) and got the same error as you:
```
Collecting nvidia-eff@ http://sqrl.nvidia.com/dldata/pip-scratch/nvidia-eff/nvidia_eff-0.1.0-py3-none-any.whl
  WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPConnection object at 0x7ff14a589050>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /dldata/pip-scratch/nvidia-eff/nvidia_eff-0.1.0-py3-none-any.whl
```
Ok, this is bad...

Reply:
yes, please, let's remove from reqs until generally available

Reply:
ok, removed the dependency, thanks @blisc for catching that...! 👍